projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-related-posts.php

```diff
@@ -34,7 +34,7 @@ public function register_routes() {
 				'methods'             => WP_REST_Server::READABLE,
 				'callback'            => array( $this, 'get_options' ),
```

Comment:
I can see why someone used `manage_option` cap since in callback we `get_options`. Assuming this will continue work regardless with the `edit_posts` cap, tho!